modules/kubernetes/pages/k8s-operator/backup-and-restore.adoc

```diff
@@ -0,0 +1,769 @@
+----
+
+Then restore the backup:
+
+[source.wrap,console]
+----
+kubectl tg restore --namespace tigergraph --cluster-name test-cluster \
+--tag tests3-2022-10-31T031005 \
+--source s3Bucket --s3-bucket tg-backup \
+--aws-secret aws-secret
+----
+
+=== Cross-cluster restore
+
+Cross-cluster restoration is when you restore a existing cluster from a backup created by another cluster.
+
+Assume that you have created a backup for a cluster called `source-cluster`, and the backup is stored in an S3 bucket.
+Your target cluster, here named `target-cluster` must have the same size, HA replication factor, and version as `source-cluster`.
+
+First, get the metadata of the backup for `source-cluster`. This command stores the data printed to the console in a file called `backup-metadata` (no extension).
+
+[source.wrap, console]
+----
+kubectl tg backup list --cluster-name source-cluster --namespace tigergraph \
+--tag tests3-2022-10-31T031005 --meta > backup-metadata
+----
+
+Then, if you haven't already, create a secret in Kubernetes containing the access key ID and secret key:
+
+[source.wrap, console]
+----
+kubectl create secret generic aws-secret \
+--from-literal=accessKeyID=AWSACCESSKEY \
+--from-literal=secretAccessKey='AWSSECRETKEY'
+----
+
+Then use the metadata and secret to restore `target-cluster`.
+
+[source.wrap, console]
+----
+kubectl tg restore --namespace tigergraph --cluster-name target-cluster \
+  --metadata backup-metadata \
+  --source s3Bucket --s3-bucket tg-backup \
+  --aws-secret aws-secret
+----
+
+=== Clone a cluster
+You can create a clone (snapshot copy) of a cluster by creating an empty new cluster and then restoring  it from a backup created by the original cluster.
+
+Assume that you have created a backup for `source-cluster`, and the backup is stored in an S3 bucket.
+You want to “clone” the cluster to a new cluster that has no existing data.
+
+The `kubectl tg restore` command can help create a new cluster based on the configuration of `source-cluster` and restore it from the backup.
+
+First get the CR of the cluster.
+
+[source.wrap, console]
+----
+kubectl tg export --cluster-name source-cluster -n tigergraph
+[source.wrap, console]
+----
+For this example, we assume the output file is located at /home/test-cluster_backup_1668069319.yaml
+
+Next get the backup metadata.
+
+[source.wrap, console]
+----
+kubectl tg backup list --cluster-name source-cluster --namespace tigergraph\
+--tag tests3-2022-10-31T031005 --meta > backup-metadata
+----
+
+Use the cluster template and metadata to create a copy of the source cluster.
+
+[source.wrap, console]
+----
+ kubectl tg restore --namespace tigergraph --cluster-name new-cluster \
+  --metadata backup-metadata --cluster-template /home/test-cluster_backup_1668069319.yaml \
+  --source s3Bucket --s3-bucket tg-backup --aws-access-key-id AWS_ACCESS_KEY \
+  --aws-secret-access-key AWS_SECRET
+----
+
+A new cluster named new-cluster will be created and initialized.
```

Comment:
Whe4n referencing particular test that had been in a code snippet, should we use code style?
That is, should new-cluster be in code style?

Reply:
I am a little confused about this comment. But for using kubectl tg commands, we do not have a specific coding style.

Reply:
Victor is referring to using the backtick ` to put the text in the doc into the monospace code style: `new-cluster`